src/parser/core/modules/CooldownDowntime.js

```diff
@@ -59,7 +59,7 @@ export default class CooldownDowntime extends Module {
 		if (downtime < 0) {
 			downtime = 0
 		}
-		const possibleNumberOfUses = Math.floor(fightlength/(cooldown*1000))
+		const possibleNumberOfUses = Math.ceil(fightlength/(cooldown*1000))
```

Comment:
Why ceiling? This makes sense if you can pre-pull the particular CD but otherwise you'd need to round down.

Reply:
Imagine a ogcd with a 3 min cooldown. The fight is 2 min long. How often can you use said cooldown? Now imagine the fight being 4 min long. That's why I changed it